test/wild2.sh

```diff
@@ -18,7 +18,7 @@ readonly OSH=~/git/oil/bin/osh
 replace-shebang() {
   local dir=$1
   find $dir -name '*.sh' \
-    | xargs -- sed -i 's|^#!/bin/bash|#!/home/andy/git/oil/bin/osh|'
+    | xargs -- sed -i 's|^#!/usr/bin/env bash|#!/home/andy/git/oil/bin/osh|'
```

Comment:
This one should remain /bin/bash because it's actually for changing OTHER repos.  Like the toybox repo -- if I want to test out OSH then I change all the shebang lines to OSH.